scrapper.py

```diff
@@ -44,20 +78,92 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(self.article_url, self.article_id)
+
+    def _fill_article_with_text(self, article_bs):
+        pdf_url = article_bs.find('iframe', class_='pdf')['data-src']
+        pdf = PDFRawFile(pdf_url, self.article_id)
+        pdf.download()
+
+        pdf_with_lit = pdf.get_text()
+        pdf_without_lit = pdf_with_lit.split('Литература')[0]
+
+        self.article.text = pdf_without_lit
+
+    def _fill_article_with_meta_information(self, article_bs):
+        self.article.author = article_bs.find('span', class_='field__item-wrapper').text
+
+        self.article.title = article_bs.find('span', class_='field field-name-title field-formatter-string '
+                                                            'field-type-string field-label-hidden').text
+
+        node_content = article_bs.find('div', class_='node__content clearfix')
+        year = node_content.find('div').find_next_siblings()[2].text.strip('\n').strip(' ')[:4]
+        self.article.date = datetime.date(int(year), 1, 1)
+
+        topics_bs = article_bs.find_all('span', class_='field__item-wrapper')[1:]
+        self.article.topics = [topic.text for topic in topics_bs]
+
+    def parse(self):
+        response = requests.get(self.article_url)
+        article_bs = BeautifulSoup(response.text, 'lxml')
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+        return self.article
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    path_to_base_path = Path(base_path)
+    if path_to_base_path.exists():
+        shutil.rmtree(base_path)
+    path_to_base_path.mkdir(parents=True)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path) as file:
+        configuration = json.load(file)
+
+    http_pattern = re.compile(HTTP_PATTERN)
+    for url in configuration["seed_urls"]:
+        result = http_pattern.search(url)
+        if not result:
+            raise IncorrectURLError
+
+    seed_urls = configuration["seed_urls"]
```

Comment:
is it guaranteed to have `seed_urls `and `total_articles_to_find_and_parse` in a configuration file?